### PR TITLE
ADR 806- Acceptance Tests added 

### PR DESCRIPTION
**Pull request by Kavitakum**
Acceptance Tests added

### CODE REVIEW

File: src/test/resources/features/quarterlySpiritsQuestions/DeclareSpiritsTotal.feature [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@Test @ZAP @@a11y @AlcoholToDeclare
+Feature: Alcohol Duty Returns - Spirits Total
```
Comment (yasaracar) [inline src/test/resources/features/quarterlySpiritsQuestions/DeclareSpiritsTotal.feature:2]: Mention Quarterly Spirit Questions on the feature name maybe?

File: src/test/resources/features/quarterlySpiritsQuestions/DeclareSpiritsTotal.feature [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@Test @ZAP @@a11y @AlcoholToDeclare
```
Comment (yasaracar) [inline src/test/resources/features/quarterlySpiritsQuestions/DeclareSpiritsTotal.feature:1]: The file name should be QuarterlySpiritQuestions.feature

File: src/test/resources/features/quarterlySpiritsQuestions/DeclareSpiritsTotal.feature [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+Feature: Alcohol Duty Returns - Spirits Total
+
+  Background: : Common Steps - Alcohol Duty Returns Journey
+    Given I navigate to the "Auth Login Stub Page"
+    When I enter redirectURL on "Auth Login Stub Page"
+    And I click submit button on "Auth Login Stub Page"
+
+  Scenario: Alcohol Duty Returns - Declare Spirits Total Page
```
Comment (yasaracar) [inline src/test/resources/features/quarterlySpiritsQuestions/DeclareSpiritsTotal.feature:9]: This Scenario is the happy path for quarterly spirit questions journey. Rename please

File: src/test/resources/features/quarterlySpiritsQuestions/SpiritsTotalErrorMessages.feature [hunk 4]
```diff
@@ -0,0 +1,37 @@
+@Test @AlcoholToDeclare
+Feature: Alcohol Duty Returns Journey - Error Messages
```
Comment (yasaracar) [inline src/test/resources/features/quarterlySpiritsQuestions/SpiritsTotalErrorMessages.feature:2]: Change the file name and feature name for this as well please

File: src/test/resources/features/quarterlySpiritsQuestions/SpiritsTotalErrorMessages.feature [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+  Background: Alcohol Duty Returns Journey - Common Steps
```
Comment (yasaracar) [inline src/test/resources/features/quarterlySpiritsQuestions/SpiritsTotalErrorMessages.feature:4]: Mention the Quarterly spirit questions in all necessary Feature name, background and scenarios please

File: src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/declareSpiritsTotal/DeclareSpiritsTotalPage.scala [hunk 6]
```diff
@@ -0,0 +1,40 @@
+/*
```
Comment (yasaracar) [inline src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/declareSpiritsTotal/DeclareSpiritsTotalPage.scala:1]: Create a new package for this file, it should be about quarterly spirit questions

**Review by yasaracar — CHANGES_REQUESTED** [review]

File: src/test/resources/features/quarterlySpiritsQuestions/DeclareSpiritsTotal.feature (hunk 1 above)
Comment (Kavitakum) [reply src/test/resources/features/quarterlySpiritsQuestions/DeclareSpiritsTotal.feature]: Done

File: src/test/resources/features/quarterlySpiritsQuestions/DeclareSpiritsTotal.feature (hunk 2 above)
Comment (Kavitakum) [reply src/test/resources/features/quarterlySpiritsQuestions/DeclareSpiritsTotal.feature]: Done

File: src/test/resources/features/quarterlySpiritsQuestions/SpiritsTotalErrorMessages.feature (hunk 4 above)
Comment (Kavitakum) [reply src/test/resources/features/quarterlySpiritsQuestions/SpiritsTotalErrorMessages.feature]: Done

File: src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/declareSpiritsTotal/DeclareSpiritsTotalPage.scala (hunk 6 above)
Comment (Kavitakum) [reply src/test/scala/uk/gov/hmrc/alcoholDuty/pages/alcoholDuty/declareSpiritsTotal/DeclareSpiritsTotalPage.scala]: Done

**Review by yasaracar — APPROVED** [review]

**Pull request merged**